Traceback (most recent call last):
  File "main.py", line 70, in <module>
    main(DEBUG=DEBUG)
  File "main.py", line 52, in main
    cmds.Take(cmd, DEBUG)
  File "/home/pi/ANDY/src/cmds.py", line 193, in __init__
    self.video()
  File "/home/pi/ANDY/src/cmds.py", line 218, in video
    self.vid_num = self.pic_file.readline().rstrip()
AttributeError: Take instance has no attribute 'pic_file'
